### PR TITLE
Ct 2427 filters layout

### PR DESCRIPTION
**Pull request by digitalali-moj**
## Description
Update the filters layout and filters itself to some extent to match prototype and comments on the ticket
 
## Self-review checklist

* [x] (1) Quick stakeholder demo done OR
* [x] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
 
### Screenshots
Yes but might change so will add later
image
image
image
image
image
image
image
 image
image

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2427
 
### Deployment
Hopefully n/a
 
### Manual testing instructions
View case list (homepage) and check if filters in constantina look like prototype, and check notes on ticket/comments. For Search page the filters should match but also have the exemptions filter.

### CODE REVIEW

**Review by Polsonby — COMMENTED** [review]
LGTM so far :) 

The filter button doesn't seem to do any filtering right now, still working on it?

**Comment by digitalali-moj** [comment]
> LGTM so far :)
> 
> The filter button doesn't seem to do any filtering right now, still working on it?



> LGTM so far :)
> 
> The filter button doesn't seem to do any filtering right now, still working on it?

--- Not started that yet, once the layout is finalised and tested then I can pair with someone on that, unless it's something too complicated :)

(done)

**Comment by digitalali-moj** [comment]
Reminder: We need to write tests for the new Branston filters before this is complete
(Done)

File: spec/features/cases/search_filter_journeys_spec.rb [hunk 1]
```diff
@@ -251,11 +251,6 @@
 
       cases_search_page.filter_on_exemptions(common: %w{ s40 } )
 
-      cases_search_page.filter_tab_links.assigned_to_tab.click
-      cases_search_page.assigned_to_filter_panel.business_unit_search_term.set('foi')
-      cases_search_page.assigned_to_filter_panel.foi_responding_team_checkbox.click
-      cases_search_page.assigned_to_filter_panel.apply_filter_button.click
-
```
Comment (Polsonby) [inline spec/features/cases/search_filter_journeys_spec.rb]: @digitalali-moj Do you happen to recall why this stuff got removed?

Comment (digitalali-moj) [reply spec/features/cases/search_filter_journeys_spec.rb]: For everyone: Part of the ticket was to remove the assigned to filter.

File: spec/features/cases/list_filter_journey_spec.rb [hunk 2]
```diff
@@ -159,6 +159,13 @@
     scenario 'clearing individual filters', js: true do
       expect(SearchQuery.count).to eq 5
 
+      expect(open_cases_page.search_results_count.text).to eq '1 case found'
+      expect(open_cases_page.filter_crumb_for('FOI - Standard'     )).to be_present
+      expect(open_cases_page.filter_crumb_for('Trigger'            )).to be_present
+      expect(open_cases_page.filter_crumb_for('In time'            )).to be_present
+      expect(open_cases_page.filter_crumb_for('Ready to close'     )).to be_present
+      expect(open_cases_page.filter_crumb_for(@deadline_filter_text)).to be_present
```
Comment (mseedat-moj) [inline spec/features/cases/list_filter_journey_spec.rb]: No need to try and align the code on the right hand side as well, looks odd

**Review by mseedat-moj — COMMENTED** [review]
Well done, good work - recommend revising the CSS file layouts, naming and use of ID attributes.

File: spec/features/cases/list_filter_journey_spec.rb (hunk 2 above)
Comment (digitalali-moj) [reply spec/features/cases/list_filter_journey_spec.rb]: I agree, was just keeping it consistent with the rest, but happy to introduce different style to this copy/paste/change code

**Review by mseedat-moj — APPROVED** [review]

**Review by mseedat-moj — APPROVED** [review]
Minor updates after a second review re: BEM styling for CSS

**Pull request merged**